src/components/Text/styles.css.ts

```diff
@@ -0,0 +1,24 @@
+import { style } from '@vanilla-extract/css';
+import theme from '~/theme/theme.css';
+
+export default {
```

Comment:
maybe it should be an object with fields like `text` and `variant` ?

Reply:
Ok